webpack/scenes/AlternateContentSources/Details/ACSExpandableDetails.js

```diff
@@ -200,6 +201,17 @@ const ACSExpandableDetails = () => {
               <InactiveText text="N/A" />
                             }
             </List>
+            <TextContent className="margin-0-24">
+              <TextListItem component={TextListItemVariants.dt}>
```

Comment:
I have a PR up: https://github.com/Katello/katello/pull/10293 to add proper styling to some of these expandable sections so I am not worried about how it looks here. I will rebase and get that sorted as part of my PR as that also introduces some CSS classes not available in master currently. Wrapping the TextListItem inside TextList will resolve the newline issue.
```suggestion
          <TextList component={TextListVariants.dl}> 
              <TextListItem component={TextListItemVariants.dt}>
```

Reply:
Committed!